opal/mca/common/cuda/common_cuda.c

```diff
@@ -774,6 +786,11 @@ static int mca_common_cuda_stage_three_init(void)
                             "CUDA: cuMemHostRegister OK on test region");
     }
 
+#if OPAL_CUDA_GPU_MEM_WAR
```

Comment:
Same logic as above applies to this `#if` (and the `#if` below): if you make it 100% runtime, it might be a little less confusing when one user says "Hey, I have this CUDA MCA param," but another user says "Hey, why don't I have that same CUDA MCA param?  We have the same versions of OMPI, right?"